10.1/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -92,6 +94,7 @@ function initialize_database() {
   # Using empty --basedir to work-around https://bugzilla.redhat.com/show_bug.cgi?id=1406391
   mysql_install_db --rpm --datadir=$MYSQL_DATADIR --basedir=''
   start_local_mysql "$@"
+  mysql_upgrade --socket=/tmp/mysql.sock
```

Comment:
That's something I found during investigation, that `mysql_upgrade` also marks data properly with a version, so we can either add it manually or let `mysql_upgrade` do it, while it also checks the data dir, which in empty (freshly initialize) case is very quick and might find potential issues (which is not expected, but one never knows).